calico-vpp-agent/cni/network_vpp.go

```diff
@@ -414,6 +414,8 @@ func (s *Server) AddVppInterface(args *pb.AddRequest, doHostSideConf bool) (ifNa
 		MacAddress:     vppSideMacAddress,
 		HostMacAddress: containerSideMacAddress,
 		RxQueues:       config.TapRXQueues,
+		RxRingSize:     config.TaxRxRingSize,
+		TxRingSize:     config.TaxTxRingSize,
```

Comment:
Typo?

Reply:
Indeed, didn't see it at first glance, thanks 😄

Reply:
I'll do the taxreform then 😛

Reply:
😃